diff -up libppd-2.0.0/Makefile.am.omv~ libppd-2.0.0/Makefile.am
--- libppd-2.0.0/Makefile.am.omv~	2023-09-24 15:53:01.916992297 +0200
+++ libppd-2.0.0/Makefile.am	2023-09-24 15:53:08.788423959 +0200
@@ -120,7 +120,7 @@ libppd_la_SOURCES = \
 	$(pkgppddefs_DATA)
 libppd_la_LIBADD = \
 	$(LIBCUPSFILTERS_LIBS) \
-	$(CUPS_LIBS)
+	$(CUPS_LIBS) -lz
 libppd_la_CFLAGS = \
 	$(LIBCUPSFILTERS_CFLAGS) \
 	$(CUPS_CFLAGS)
diff -up libppd-2.0.0/ppd/rastertops.c.omv~ libppd-2.0.0/ppd/rastertops.c
